client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx

```diff
@@ -87,9 +89,11 @@ export default function SiteRow( {
 		SubscriptionManager.useSiteEmailMeNewCommentsMutation();
 	const { mutate: unsubscribe, isLoading: unsubscribing } =
 		SubscriptionManager.useSiteUnsubscribeMutation();
+	const { mutate: resubscribe, isLoading: resubscribing } =
+		SubscriptionManager.useSiteSubscribeMutation();
 
 	return (
-		<li className="row" role="row">
+		<li className={ classnames( 'row', { deleted: isDeleted } ) } role="row">
```

Comment:
With the move to a notification popup, could we instead, reuse the `isDeleted` flag so that it doesn't render the list item?